src/main/resources/native/android/dalvik/KeyCode.java

```diff
@@ -0,0 +1,1331 @@
+     */
+    POWER(0x199, "Power"),
+
+    /**
+     * Constant for the {@code Info} key.
+     */
+    INFO(0x1C9, "Info"),
+
+    /**
+     * Constant for the {@code Colored Key 0} key.
+     */
+    COLORED_KEY_0(0x193, "Colored Key 0"),
+
+    /**
+     * Constant for the {@code Colored Key 1} key.
+     */
+    COLORED_KEY_1(0x194, "Colored Key 1"),
+
+    /**
+     * Constant for the {@code Colored Key 2} key.
+     */
+    COLORED_KEY_2(0x195, "Colored Key 2"),
+
+    /**
+     * Constant for the {@code Colored Key 3} key.
+     */
+    COLORED_KEY_3(0x196, "Colored Key 3"),
+
+    /**
+     * Constant for the {@code Eject} key.
+     */
+    EJECT_TOGGLE(0x19E, "Eject", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Play} key.
+     */
+    PLAY(0x19F, "Play", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Record} key.
+     */
+    RECORD(0x1A0, "Record", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Fast Forward} key.
+     */
+    FAST_FWD(0x1A1, "Fast Forward", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Rewind} key.
+     */
+    REWIND(0x19C, "Rewind", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Previous Track} key.
+     */
+    TRACK_PREV(0x1A8, "Previous Track", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Next Track} key.
+     */
+    TRACK_NEXT(0x1A9, "Next Track", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Channel Up} key.
+     */
+    CHANNEL_UP(0x1AB, "Channel Up", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Channel Down} key.
+     */
+    CHANNEL_DOWN(0x1AC, "Channel Down", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Volume Up} key.
+     */
+    VOLUME_UP(0x1bf, "Volume Up", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Volume Down} key.
+     */
+    VOLUME_DOWN(0x1C0, "Volume Down", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the {@code Mute} key.
+     */
+    MUTE(0x1C1, "Mute", KeyCodeClass.MEDIA),
+
+    /**
+     * Constant for the Apple {@code Command} key. 
+     * @since JavaFX 2.1
+     */
+    COMMAND(0x300, "Command", KeyCodeClass.MODIFIER),
+    
+    /**
+     * Constant for the {@code Shortcut} key.
+     */
+    SHORTCUT(-1, "Shortcut");
+
+    final int code;
+    final String ch;
+    final String name;
+    private int mask;
+
+    // Need to bundle this in another class to avoid "forward reference" compiler error
+    private static class KeyCodeClass {
+        private KeyCodeClass() {}
+
+        private static final int FUNCTION = 1;
+        private static final int NAVIGATION = 1 << 1;
+        private static final int ARROW = 1 << 2;
+        private static final int MODIFIER = 1 << 3;
+        private static final int LETTER = 1 << 4;
+        private static final int DIGIT = 1 << 5;
+        private static final int KEYPAD = 1 << 6;
+        private static final int WHITESPACE = 1 << 7;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4825537)